api/src/services/files.ts

```diff
@@ -86,37 +84,30 @@ export class FilesService extends ItemsService {
 				payload.height = meta.height;
 			}
 
-			payload.filesize = meta.size;
```

Comment:
I've removed this line because payload.filesize is already set on line 73, no need to overwrite it again.